pkg/mount/mountinfo_unsupported.go

```diff
@@ -1,4 +1,4 @@
-// +build !windows,!linux,!freebsd freebsd,!cgo
+// +build !windows,!linux,!freebsd, !cgo darwin
```

Comment:
is it okay to delete freebsd related code here ?

Reply:
I am not sure, but felt like `!freebsd freebsd` this didn't make sense. I can revert it back.

Reply:
pls